plugin/http/httptrace/httptrace.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package httptrace contains OpenCensus tracing integrations with net/http.
+package httptrace
+
+import (
+	"net/http"
+	"strings"
+
+	"go.opencensus.io/trace"
+	"go.opencensus.io/trace/propagation"
+)
+
+const httpHeader = `X-Cloud-Trace-Context`
+
+type transport struct {
+	base        http.RoundTripper
+	propogators []propagation.HTTPPropagator
```

Comment:
Do we want to encourage people to add multiple propagators? Seems like it would be more common to standardize on a particular format or perhaps to accept multiple formats but produce only a single format.